meilisearch/tests/client/test_client_dumps.py

```diff
@@ -0,0 +1,44 @@
+import json
+import pytest
+import meilisearch
+from meilisearch.tests import BASE_URL, MASTER_KEY, clear_all_indexes, wait_for_dump_creation
+from meilisearch.errors import MeiliSearchApiError
+
+class TestClientDumps:
+
+    """ TESTS: Client dumps creation and status """
+
+    client = meilisearch.Client(BASE_URL, MASTER_KEY)
+    index = None
+    dataset_file = None
+    dataset_json = None
+
+    def setup_class(self):
+        clear_all_indexes(self.client)
+        self.index = self.client.create_index(uid='indexUID')
+        self.dataset_file = open('./datasets/small_movies.json', 'r')
+        self.dataset_json = json.loads(self.dataset_file.read())
+        self.dataset_file.close()
+
+    def test_dump_creation(self):
+        """Tests the creation of a MeiliSearch dump"""
+        dump = self.client.create_dump()
+        assert dump['uid'] is not None
+        assert dump['status'] == 'processing'
```

Comment:
I think like @curquiza said you should add a `wait_for_dump_creation` here. I think the reason you did not is because you already wait for `add_documents` to be finished in the next test. Since test could be added and/or changed I suggest adding `wait_for_dump_creation` here to be sure that whatever the next test is it will not fail because dump creation is not done.